lib/exposes.js

```diff
@@ -430,10 +430,10 @@ module.exports = {
         light_brightness: () => new Light().withBrightness(),
         light_brightness_color: () => new Light().withBrightness().withColor((['xy', 'hs'])),
         light_brightness_colorhs: () => new Light().withBrightness().withColor(['hs']),
-        light_brightness_colortemp: () => new Light().withBrightness().withColorTemp().withColorTempStartup(),
-        light_brightness_colortemp_color: () => new Light().withBrightness().withColorTemp().withColorTempStartup().withColor(['xy', 'hs']),
-        light_brightness_colortemp_colorhs: () => new Light().withBrightness().withColorTemp().withColorTempStartup().withColor(['hs']),
-        light_brightness_colortemp_colorxy: () => new Light().withBrightness().withColorTemp().withColorTempStartup().withColor(['xy']),
+        light_brightness_colortemp: (colorTempRange) => new Light().withBrightness().withColorTemp(colorTempRange).withColorTempStartup(colorTempRange),
```

Comment:
@Koenkk I assume it's failing here, because this is colorTempRange is not passed because this isn't a function as I first thought... any idea on how to cleanly pass these down ?

Reply:
Why do you think this doesn't work? Looks ok here.

```
const devices = require('../devices');

const d = devices.find(d => d.model === '9290012573A');
console.log(d.exposes[0].features);
```

outputs

```
...
  Numeric {
    type: 'numeric',
    name: 'color_temp',
    property: 'color_temp',
    access: 7,
    unit: 'mired',
    value_min: 153,
    value_max: 500,
    description: 'Color temperature of this light'
  },
  Numeric {
    type: 'numeric',
    name: 'color_temp_startup',
    property: 'color_temp_startup',
    access: 7,
    unit: 'mired',
    value_min: 153,
    value_max: 500,
    description: 'Color temperature after cold power on of this light',
    presets: [ [Object] ]
  },
...
```

Reply:
Tweaked my debug logs, it is indeed working.
Will do some more testing and mark this as Ready.